elyra/templates/components/generic_properties_template.jinja2

```diff
@@ -35,16 +35,28 @@
             "items": []
           }
         },
-        "cpu": {
+        "cpu_request": {
```

Comment:
I don't think we can change the names of _existing_ pipeline fields w/o introducing a migration.  As a result, I would recommend retaining the names `cpu` and `memory` and only introducing `cpu_limit` and `memory_limit`.

(See the test pipeline files.  Any changes to field name will require a migration.)

Reply:
I have changed all the test pipelines files too I believe, what would the migration involve?

Reply:
This doesn't address all the existing pipeline files that still use `cpu`.  Those pipelines will be loaded but the cpu (and memory) request value(s) will not be found.  I really don't think we should introduce a migration for this but instead update the `description` entries to emphasize these are _request_ values.  The UI will still reflect proper naming, only the pipeline file field entries would remain `cpu` and `memory`.

> what would the migration involve?

We'd need to increment the pipeline version and when an older version is detected, read the `cpu` and `memory` fields into `cpu_request` and `memory_request` and ensure the updated pipeline is persisted.  There is a framework for this in place already although I have never worked with it.

Reply:
I might not have access to those older pipelines, so I have reverted the request names to cpu and memory to avoid any issues

Reply:
Pipeline files (`.pipeline`) are tantamount to notebook files (`.ipynb`) and will exist wherever elyra has been installed and used to create a pipeline.  This is why any changes to the "file format" need to be well-intentioned.  I'm not saying this PR doesn't warrant an update to file format, but that update introduces more moving parts (and points for error) than is probably worth the effort.  

Thank you for your patience on this pull request.